src/components/outsourcing/CustomerConcerns.jsx

```diff
@@ -1,7 +1,26 @@
 import React from "react";
```

Comment:
There is currently no CSS (aka there is no styling, there is only html). You need to add some tailwindCSS to this component so it looks like the figma. 

So making the text bold, adjusting the fonts, etc